packages/infra/lib/api-stack/api-service.ts

```diff
@@ -351,6 +351,21 @@ export function createAPIService({
 
   // Access grant for Aurora DB's secret
   dbCredsSecret.grantRead(fargateService.taskDefinition.taskRole);
+  // Access to EHR secrets
+  if (props.config.ehrIntegration) {
+    const athenaClientKeySecret = secret.Secret.fromSecretCompleteArn(
+      stack,
+      "EhrAthenaClientKeySecret",
+      props.config.ehrIntegration.athenaHealth.athenaClientKeyArn
+    );
+    athenaClientKeySecret.grantRead(fargateService.taskDefinition.taskRole);
+    const athenaClientSecretSecret = secret.Secret.fromSecretCompleteArn(
+      stack,
+      "EhrAthenaClientSecretSecret",
+      props.config.ehrIntegration.athenaHealth.athenaClientSecretArn
+    );
+    athenaClientSecretSecret.grantRead(fargateService.taskDefinition.taskRole);
```

Comment:
I don't think the "Secret" suffix helps to make it readable (its always in the context of secrets anyways), but just a nit.

Reply:
Done.